template.yaml

```
@@ -1243,6 +1243,10 @@ Outputs:
   S3CopyAndEncryptLogsName:
     Value: !Ref S3CopyAndEncryptLogs
 
-  # Tempory Audit S3 Bucket
-  TemporaryMessageBatchBucketName:
-    Value: !Ref TemporaryMessageBatchBucket
+  # S3 Temp Audit Bucket
+  AuditMessageBatchName:
```

Can we call this output:
```suggestion
  AuditMessageBatchBucketName:
```

Just so it's clear it's a bucket :)